gym/spec/build_command_generator_spec.rb

```diff
@@ -22,7 +22,7 @@
       xcargs = xcargs_hash.map do |k, v|
         "#{k.to_s.shellescape}=#{v.shellescape}"
       end.join ' '
-      options = { project: "./examples/standard/Example.xcodeproj", sdk: "9.0", xcargs: xcargs, scheme: 'Example' }
+      options = { project: "./examples/standard/Example.xcodeproj", sdk: "9.0", toolchain: "com.apple.dt.toolchain.Swift_2_3", xcargs: xcargs, scheme: 'Example' }
```

Comment:
Does the toolchain always start with `com.apple.dt.toolchain`? If so, we could automatically prefix user values 👍